ipn/ipnlocal/local_test.go

```diff
@@ -3772,7 +3772,7 @@ func TestLocalBackendSuggestExitNode(t *testing.T) {
 		lb := newTestLocalBackend(t)
 		lb.lastSuggestedExitNode = tt.lastSuggestedExitNode
 		lb.netMap = &tt.netMap
-		lb.sys.MagicSock.Get().SetLastNetcheckReport(context.Background(), tt.report)
+		lb.sys.MagicSock.Get().SetLastNetcheckReportForTest(context.Background(), &tt.report)
```

Comment:
minor, but I'd move the pointer up to the field. We try to stay in as-a-value vs as-a-pointer consistently for each type and netcheck.Report is always used as a pointer (as it's super big: https://go.dev/wiki/CodeReviewComments#receiver-type)